src/lib/reasoners/bitv.ml

```diff
@@ -198,8 +198,10 @@ module Shostak(X : ALIEN) = struct
       | Vother of 'a
       | Vextract of 'a * int * int
       | Vconcat of 'a * 'a
+      | Vnot of 'a
+      (*     [@@deriving show] *)
```

Comment:
Either remove this and do not add `ppx_deriving` as a dependency, or keep `ppx_deriving` as a dependency and uncomment this. I'm not too keen on adding `ppx_deriving` for `show` implementations (I find that the output is usually barely readable, due to printing "raw" OCaml ASTs), but at the very least let's not add an unused dependency :)

Reply:
Actually, I use this dependency and I plan to use it in the codebase later. In the present code, I don't keep the deriving attribute because it produces a warning if we don't use the print function. I used these print functions only for debugging.

Reply:
What I mean is that if there is no *code* that actually uses the dependency, it shouldn't be a dependency. If the issue is that we want to use it but it generates "unused code" warning, let's silence warning 32 locally.